components/feature/prompts/src/test/java/mozilla/components/feature/prompts/creditcard/CreditCardPickerTest.kt

```diff
@@ -105,6 +112,32 @@ class CreditCardPickerTest {
         verify(creditCardSelectBar).showPrompt(promptRequest.creditCards)
     }
 
+    @Test
+    fun `GIVEN a selected credit card WHEN onAuthSuccess is called THEN the confirmed credit card is received`() {
+        assertNull(creditCardPicker.selectedCreditCard)
```

Comment:
Could we also confirm the prompt request was consume and confirmed? :)

Reply:
We do that by `assertEquals(creditCard, confirmedCreditCard)` check below

Reply:
Thanks for the clarification, as this value is shared for multiple tests, could be reset it on `setup()` 👍🏽